FORMULA/crowdin.rb

```diff
@@ -0,0 +1,16 @@
+class Crowdin < Formula
+  desc "Crowdin-CLI is a command-line tool that allows to manage your resources"
+  homepage "https://support.crowdin.com/cli-tool/"
+  url "https://crowdin.com/downloads/crowdin-cli-2.0.11.zip"
+  sha256 "027925176948b8973ba1a9fab4f0c097bf49f2df0c23be20d79049bd7af4fbc3"
+
+  def install
+    bin.install "crowdin-cli.jar"
+    system "alias", "crowdin=\"'java -jar #{bin}/crowdin-cli.jar'\" >> ~/.bashrc"
+    system "alias", "crowdin=\"'java -jar #{bin}/crowdin-cli.jar'\" >> ~/.bash_profile"
+  end
+
+  test do
+    system "java", "-jar", "#{bin}/crowdin-cli.jar"
```

Comment:
This is a simple test to ensure that the program is installed and working correctly. The next version will add more tests.